metrics/web/docs/features/metrics_logger/diagrams/metrics_logger_class_diagram.puml

```diff
@@ -0,0 +1,29 @@
+@startuml metrics_logger_class_diagram
+
+package metrics {
+  class MetricsApp {}
+
+  class LoggerWriter {
+    + writeError(Object error, [Object stackTrace]) : FutureOr<void>
+    + setContext(String key, Map<String, dynamic> context) : FutureOr<void>
+  }
+
+  class MetricsLogger {
+    - _loggerWriter : LoggerWriter
+    + static initialize(LoggerWriter loggerWriter,  Map<String, dynamic> contexts) : Future<void>
+    + static logError(Object error, [Object stackTrace]) : Future<void>
```

Comment:
Can we specify stackTrace type to be StackTrace? Can you ping me so we can discuss this - I feel that I miss something here.